src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -2653,9 +2649,7 @@ void CEEInfo::embedGenericHandle(
 
     _ASSERTE(pResult->compileTimeHandle);
 
-    if (fRuntimeLookup
-            // Handle invalid IL - see comment in code:CEEInfo::ComputeRuntimeLookupForSharedGenericToken
```

Comment:
The handling of Invalid IL here was a desktop compat quirk. No reason to keep it around.